docs/mcd.md

```diff
@@ -1,5 +1,6 @@
-# Connection Dialog v2
-
+---
+title: Connection Dialog v2
+--- 
 **IMPORTANT:** This is alpha software and should be used in a test environment. Do not deploy the connector to a production environment.
```

Comment:
Remove this alpha warning.

Reply:
Should probably also add a line about it being 2020.3+ only. As prominent as possible, so people don't miss it and wonder why their connector is breaking in old versions.